deebot_client/events/__init__.py

```diff
@@ -308,3 +315,11 @@ class FirmwareEvent(Event):
     """Firmware event."""
 
     version: str
+
+
+@dataclass(frozen=True)
+class RainDelayEvent(Event):
+    """Rain delay event representation."""
+
+    enable: bool
```

Comment:
```suggestion
    enabled: bool
```

Reply:
Ok, so I'll remove the new commands here and this MR is just a copy of the Goat G1 for the Goat R1600.